.goreleaser.yml

```diff
@@ -0,0 +1,61 @@
+before:
+  hooks:
+    - make release
+builds:
+  -
+    id: capp
+    env:
+      - CGO_ENABLED=0
+    binary: manager
+    goos:
+      - darwin
+      - linux
+    goarch:
+      - amd64
+      - arm64
+release:
+  extra_files:
+    - glob: ./out/release/infrastructure-packet/v0.0.0/*
```

Comment:
I don't understand, the version is 0.0.0?

Reply:
ah this has to be fixed 👍 it will be `{ .Tag }`

Reply:
OK